autoload/coqtail.vim

```diff
@@ -402,10 +447,66 @@ function! coqtail#start(...) abort
   return 1
 endfunction
 
+" Callback to be run after Coqtop has launched.
+function! coqtail#after_startCB(chan, msg) abort
+  call s:unlock_buffer(a:msg.buf)
+
+  " l:buf is the number of the current buffer
+  let l:buf = bufnr('')
+  " Hack: switch to the buffer that is running this Coq instance
+  execute 'noautocmd keepalt buffer' a:msg.buf
+
+  let l:ret_msg = a:msg.ret
+  " l:ret_msg is [coqtail_error_message, coqtop_stderr]
+  if l:ret_msg[0] != v:null
+    let l:msg = 'Failed to launch Coq.'
+    let l:msg .= "\n" . l:ret_msg[0]
+    if l:ret_msg[1] !=# ''
+      let l:msg .= "\n" . l:ret_msg[1]
+    endif
+    call coqtail#util#err(l:msg)
+    call coqtail#stop()
+    return 0
+  endif
+
+  call coqtail#refresh()
+
+  call s:init_proof_diffs(b:coqtail_version.str_version)
+
+  " Call the after_start_cmd, if present
+  if b:after_start_cmd != v:null
+    execute b:after_start_cmd
```

Comment:
`b:after_start_cmd` should be reset to `v:null` or deleted with `unlet`.

Reply:
Really it should be passed as an argument to the callback, but the way `s:call` works currently doesn't make that very easy, so this is fine for now.